demo_order_by_angle.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
```

Comment:
Maybe we should ignore this demo for the main branch? We should discuss what demos we want so we do not overflow the repo with a lot of demos xD.